pyro/infer/mcmc/mcmc.py

```diff
@@ -268,11 +268,23 @@ def _traces(self, *args, **kwargs):
             yield sample
 
     def marginal(self, sites=None):
+        """
+        Marginalizes latent sites from the sampler.
+
+        :param list sites: optional list of sites for which we need to generate
+            the marginal distribution.
+        :returns: A :class:`MCMCMarginals` class instance.
+        :rtype: :class:`MCMCMarginals`.
+        """
         return MCMCMarginals(self, sites)
 
 
 class MCMCMarginals(Marginals):
     def diagnostics(self):
+        """
+        Gets some diagnostics statistics such as effective number size and
```

Comment:
"effective sample size" is clearer imo

Reply:
Thanks @jpchen! I'll address them soon. :)